README.md

```diff
@@ -78,23 +89,28 @@ _NOTE_: If you use a Mac, get the corresponding `*.dylib` files
 
 ### Obtain Certificate
 
-You also need to obtain a test certificate from ELSTER and place it under `erica/instances/blueprint/cert.pfx`.
+You also need to obtain a test certificate from ELSTER and place it under `erica/erica_legacy/instances/blueprint/cert.pfx`.
 
 ## Developing 👩‍💻 👨‍💻
 
-
-Start your docker:
+### Start your docker:
 ```bash
 docker-compose up
 ```
 
+### Start the erica API:
 ```bash
+pipenv shell
 export ERICA_ENV=development
-python -m erica 
+export SQLALCHEMY_DATABASE_URI=postgresql://postgres:postgres@localhost/db 
+alembic upgrade head
```

Comment:
I think the alembic migration should not be part of the "Start the erica API". Maybe another category before "Start the erica API" maybe called "Run Alembic migration"? Can also be done in 1 line:
`env ERICA_ENV=development SQLALCHEMY_DATABASE_URI=postgresql://postgres:postgres@localhost/db alembic upgrade head`

Reply:
Done!